libparsec/crates/types/tests/unit/id.rs

```diff
@@ -38,3 +38,20 @@ fn human_handle(#[case] email: &str, #[case] label: &str, #[case] is_ok: bool) {
         is_ok
     );
 }
+
+#[rstest]
+#[case::alphanum("01239AbcdDEFA", true)]
+#[case::with_underscore("hello_World", true)]
+#[case::with_hyphen("hello-World", true)]
+// cspell:disable-next-line
+#[case::to_long("aaaabbbbccccddddeeeeffffgggghhhhiiiijjjjj", false)]
```

Comment:
```suggestion
#[case::too_long("aaaabbbbccccddddeeeeffffgggghhhhiiiijjjjj", false)]
```

Reply:
and also, this is 41 characters long (so 41bytes), would be better to have one test failing with 33bytes and another one passing with 32bytes

Reply:
done